app/src/org/commcare/dalvik/activities/CommCareSetupActivity.java

```diff
@@ -664,6 +666,11 @@ protected void deliverError(CommCareSetupActivity receiver, Exception e) {
                     break;
                 }
             }
+            // attemptedInstall will only be true if we found no texts with the SMS_INSTALL_KEY_STRING tag
+            // if we found one, notification will be handle by the task receiver
+            if(!attemptedInstall) {
+                Toast.makeText(this, Localization.get("menu.sms.not.found"), Toast.LENGTH_LONG).show();
+            }

```

Comment:
Probably only want to show this if `installTriggeredManually` is `true`. 
So `if (!attemptedInstall && installTriggeredManually)`